Resources/Private/Language/locallang.xlf

```diff
@@ -9,10 +9,10 @@
 			<trans-unit id="plugin.tea.view">
```

Comment:
This label now is unused and can be removed (also in the DE locallang file).

Reply:
We also need to check whether the `plugin.tea` key is still used or whether it can be removed.

Reply:
There are no more usage of plugin.tea, so I removed it.